bundle/phases/initialize.go

```diff
@@ -77,42 +122,136 @@ func Initialize(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+
+		// Reads (dynamic): resources.dashboards.* (checks for existing parent_path and embed_credentials)
+		// Updates (dynamic): resources.dashboards.*.parent_path (sets to workspace.resource_path if not set)
+		// Updates (dynamic): resources.dashboards.*.embed_credentials (sets to false if not set)
 		mutator.ConfigureDashboardDefaults(),
+
+		// Reads (dynamic): resources.volumes.* (checks for existing volume_type)
+		// Updates (dynamic): resources.volumes.*.volume_type (sets to "MANAGED" if not set)
 		mutator.ConfigureVolumeDefaults(),
+
+		// Reads (typed): b.Config.Bundle.Mode, b.Config.Workspace.{RootPath,FilePath,ResourcePath,ArtifactPath,StatePath}, b.Config.Workspace.CurrentUser (validates paths and user info)
+		// Updates (typed): b.Config.Bundle.Deployment.Lock.Enabled, b.Config.Presets.{NamePrefix,Tags,JobsMaxConcurrentRuns,TriggerPauseStatus,PipelinesDevelopment} (configures development mode settings)
+		// Validates and configures bundle settings based on target mode (development or production)
 		mutator.ProcessTargetMode(),
+
+		// Reads (typed): b.Config.Presets.{NamePrefix,Tags,JobsMaxConcurrentRuns,TriggerPauseStatus,PipelinesDevelopment} (reads preset configurations)
+		// Updates (typed): b.Config.Resources.{Jobs,Pipelines,Models,Experiments,ModelServingEndpoints,RegisteredModels,QualityMonitors,Schemas,Clusters,Dashboards} (applies prefix to names, adds tags, sets pause status and development mode)
+		// Applies all presets to resources, including name prefixes, tags, and resource-specific settings like pause status
 		mutator.ApplyPresets(),
+
+		// Reads (typed): b.Config.Resources.Jobs (checks job configurations)
+		// Updates (typed): b.Config.Resources.Jobs[].Queue (sets Queue.Enabled to true for jobs without queue settings)
+		// Enable queueing for jobs by default, following the behavior from API 2.2+.
 		mutator.DefaultQueueing(),
+
+		// Reads (dynamic): resources.pipelines.*.libraries (checks for notebook.path and file.path fields)
+		// Updates (dynamic): resources.pipelines.*.libraries (expands glob patterns in path fields to multiple library entries)
+		// Expands glob patterns in pipeline library paths to include all matching files
 		mutator.ExpandPipelineGlobPaths(),
 
+		// Reads (typed): b.SyncRoot (checks if bundle root is in /Workspace/)
+		// Updates (typed): b.SyncRoot (replaces with extension-aware path when running on Databricks Runtime)
 		// Configure use of WSFS for reads if the CLI is running on Databricks.
 		mutator.ConfigureWSFS(),
 
 		mutator.TranslatePaths(),
+
+		// Reads (typed): b.Config.Experimental.PythonWheelWrapper, b.Config.Presets.SourceLinkedDeployment (checks Python wheel wrapper and deployment mode settings)
+		// Reads (dynamic): resources.jobs.*.tasks (checks for tasks with local libraries and incompatible DBR versions)
+		// Provides warnings when Python wheel tasks require DBR 13.3+ or when wheel wrapper is incompatible with source-linked deployment
```

Comment:
This seems incorrect, the warning is for cases when python wheel tasks are used on DBRs < 13.3

Reply:
Updated.